hudi-common/src/test/java/org/apache/hudi/common/testutils/FileCreateUtils.java

```diff
@@ -307,6 +318,13 @@ public static long getTotalMarkerFileCount(String basePath, String partitionPath
         .endsWith(String.format("%s.%s", HoodieTableMetaClient.MARKER_EXTN, ioType))).count();
   }
 
+  public static List<Path> getPartitionPaths(Path basePath) throws IOException {
```

Comment:
this does not belong here? in `FileCreateUtils`? its not really creating anything

Reply:
there exists some functions too like getXXX and deleteXXX. Maybe it's time to rename this to `FileCRUDUtils`?